src/main/resources/META-INF/package.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "@openrewrite/rewrite-bundled-starter",
+  "version": "1.0.0",
+  "description": "",
+  "dependencies": {
+    "@openrewrite/rewrite-remote": "^0.3.0"
```

Thinking about it, it might be a good idea to use `~` instead. That would allow us to still roll out some fixes to the parser, which don't require changes the LST model or printer. At the same time it would allow us to create a new `0.4.0` release for whenever we want to merge changes to the LST model without causing the CLI to break. This is what I did for Python.

What do you think? The only issue I see with this is that we have to remember to update this. But since we currently have to update the `package.json` version manually anyway, it's maybe OK. Also, we can automate some of that.
```suggestion
    "@openrewrite/rewrite-remote": "~0.3.0"
```

---

Now I think I also remember that I did some automation here for Python, so that it at build time reads the current version of rewrite-remote and writes that to the bundled `requirements.txt` file. That way we don't have to worry about forgetting to update it. We could do the same thing here.